christfellowshipchurch/web-app-v2
# Encrypted Links for Background Videos

### About
This PR changes the 'http' to 'https' for the background video links

### Test Instructions
1. Go to `/locations`
2. Click on one of the location pages
3. Expect to see a lock in the search bar indicating that the field is secure

### Screenshots
Before:
<img width="611" alt="Screenshot 2023-07-13 at 2 18 43 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/bc694d57-9c2d-466f-8f8e-023f4cfdca53">
After:
<img width="703" alt="Screenshot 2023-07-13 at 3 07 45 PM" src="https://github.com/christfellowshipchurch/web-app-v2/assets/93110503/1ab9407e-6015-4cbc-956a-3cf33c919239">


### Closes Tickets
[CFDP-2618](https://christfellowshipchurch.atlassian.net/browse/CFDP-2618)



[CFDP-2618]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2618?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
